src/main/webapp/app/login/login.component.html

```diff
@@ -1,39 +1,49 @@
-<div class="loginHeader"> 
+<div class="loginHeader">
```

Comment:
I don't think it was necessary for you to change the `login.component` file... but these changes just seem to be indentations corrections. Just a note to please make sure that the login page remains functional 😇

Reply:
<3